### ISSUE TITLE
When will you release the code for pre-training 

### ISSUE THREAD

**Issue by BaohaoLiao** [microsoft/SimMIM#1]
Hi, 

the paper is great and the result is impressive. May I ask why are you going to release the code for pre-training?

**Comment by caoyue10** [comment]
The code and model will be released as soon as we complete the company process. The estimated time is two or three weeks.

**Comment by xiaohu2015** [comment]
@caoyue10 Hi, a question about the mask strategy, what value you fill the masked region, maybe [127.5, 127.5, 127.5]?
